TEAMS.md

```diff
@@ -0,0 +1,31 @@
+
+Please [reach out in any of the available ways](https://github.com/open-gitops/.github/blob/main/CONTRIBUTING.md) with any questions. We welcome contributions from [everyone](https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#all-members), and look forward to collaborating with you! 🙂💖🤝
+
+## List of teams
+
+<!-- markdownlint-disable MD033 -->
+| Team | Status | Info | Members |
+| -- | -- | -- | -- |
+| content/gitops-environmental-sustainability team | active | [current discussion](https://github.com/open-gitops/project/issues/129) | <ul><li>[@chris-short](https://github.com/chris-short)</li><li>[@JaimeMagiera](https://github.com/JaimeMagiera)</li><li>[@nikimanoledaki](https://github.com/nikimanoledaki)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@nishadmehendale](https://github.com/nishadmehendale)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@williamcaban](https://github.com/williamcaban)</li></ul> |
+| content/meme | proposed | [current discussion](https://github.com/open-gitops/project/discussions/102) | <ul><li>[scottrigby](https://github.com/scottrigby)</li></ul> |
+| content/ml | paused | [current discussion](https://github.com/open-gitops/project/issues/127) / [meeting info](https://docs.google.com/document/d/1HHkqMK_GW6dYHPu1oDjRXesPFO0u1KtyHX70FQ95_7A/edit#heading=h.m1bztpjqh35d) | <ul><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@samueltauil](https://github.com/samueltauil)</li><li>[@williamcaban](https://github.com/williamcaban)</li></ul> |
+| content/security | proposed | [current discussion](https://github.com/open-gitops/project/issues/128) | <ul><li>[roberthstrand](https://github.com/roberthstrand)</li></ul> |
+| marketing/media | active | [current discussion](https://github.com/open-gitops/project/discussions/22) | <ul><li>[@christianh814](https://github.com/christianh814)</li><li>[@chris-short](https://github.com/chris-short)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@soschwei](https://github.com/soschwei)</li><li>[@staceypotter](https://github.com/staceypotter)</li><li>[@todaywasawesome](https://github.com/todaywasawesome)</li></ul> |
+| marketing/events | active | ADD | <ul><li>[@christianh814](https://github.com/christianh814)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@roberthstrand](https://github.com/roberthstrand)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@soschwei](https://github.com/soschwei)</li><li>[@staceypotter](https://github.com/staceypotter)</li></ul> |
+| marketing/media-fact-checking | proposed | ADD | <ul><li>[@makkes](https://github.com/makkes)</li><li>[@scottrigby](https://github.com/scottrigby)</li></ul> |
+| translations | active | ADD | ADD |
+| translations/LANGUAGE | ADD | ADD | ADD |
```

Comment:
Removing for now until we set up these teams. Tracked in https://github.com/open-gitops/project/issues/149

```suggestion
```